tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/TableHandle.java

```diff
@@ -50,33 +50,40 @@ public static TableHandle parse(String value)
     private final Optional<String> schema;
     private final String name;
     private final boolean requireNoSchema;
+    private final boolean shouldCreateSchema;
```

Comment:
I don't think it is a best place for this field. TableHandle tells you what table should be used, so tempto is able to find it among different databases and schemas. It is not only used for creation but also for table selection. In case of selection this field has no sense.